softlearning/environments/adapters/dm_control_adapter.py

```diff
@@ -88,8 +93,11 @@ def __init__(self,
         assert isinstance(env.observation_spec(), OrderedDict)
         self.observation_keys = (
             observation_keys or tuple(env.observation_spec().keys()))
-        # TODO(hartikainen): Do we need to normalize actions? For now, assume
-        # we don't.
+
+        # Ensure action space is already normalized.
+        action_space_bounds = (env.action_spec().minimum[0], env.action_spec().maximum[0])
```

Comment:
Let's change this to
```python
if normalize:
    np.testing.assert_equal(env.action_spec().minimum, -1)
    np.testing.assert_equal(env.action_spec().maximum, 1)
```

Right now, we always assume that the action space dimension is 1, but I think it's better to have this test case work for other shapes as well.

Reply:
Wow that's actually pretty cool, I just tested it out and am amazed it works for multidimensional arrays with just 1 element!